testing/gearmand/gearmand.pre-install

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+## add gearmand group or use exsiting
+if ! getent group gearmand &>/dev/null; then
```

Comment:
BTW, there’s even example for this on Alpine wiki [here](https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#.24pkgname.pre-install).